modules/olm-deleting-operators-from-a-cluster-using-web-console.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * applications/operators/olm-adding-operators-to-cluster.adoc
+
+[id='olm-deleting-operators-from-a-cluster-using-web-console-{context}']
+= Deleting Operators from a cluster using the web console
+
+To delete an installed Operator from the selected namespace through the Web console, follow these steps:
```

Comment:
> You can delete an installed Operator by using the web console.
NOTE
You cannot remove an Operator from the Catalog Listing by using the web console. You must use the CLI to delete the Operator's entry in the CatalogSourceConfig resource.

Reply:
This is actually an issue that I think will go away. So, I'll just add it as a NOTE at the end so it can easily be removed later.